doc/rules/phpdoc/phpdoc_scalar.rst

```diff
@@ -13,9 +13,9 @@ Configuration
 
 A list of types to fix.
 
-Allowed values: a subset of ``['boolean', 'callback', 'double', 'integer', 'real', 'str']``
+Allowed values: a subset of ``['boolean', 'callback', 'double', 'integer', 'no-return', 'real', 'str']``
 
-Default value: ``['boolean', 'callback', 'double', 'integer', 'real', 'str']``
+Default value: ``['boolean', 'callback', 'double', 'integer', 'no-return', 'real', 'str']``
```

Comment:
changing default Fixer configuration is a bc break.

Reply:
Oh damn, good catch thanks.

Changed the config for it to not be a breaking change and make the no-return allowed but not default.

Reply:
Should be fixed now, the failing jobs seem to be failing on all MR's - and seem to not have anything to do with my changes if I get that correct.